doc/README.md

```diff
@@ -24,18 +24,18 @@ Unpack the files into a directory and run:
 
 ### Windows
 
-Unpack the files into a directory, and then run bitcoin-qt.exe.
+Unpack the files into a directory, and then run namecoin-qt.exe.
```

Comment:
I successfully built Namecoin-Qt for Linux from the Namecore source code a few months ago.  Note that I built it with Gitian.  It appeared to work fine; it synced successfully and I used it with NMControl with no problems.  I didn't try putting any NMC into the wallet though.